.workflow/mergify.yml

```
@@ -0,0 +1,137 @@
+      - merged
+    actions:
+      delete_head_branch: {}
+
+queue_rules:
+  - name: default
+    conditions:
+      - check-success=get_current_step
+      - check-success=handle_pull_request
+      - check-success=handle_review
+
+pull_request_rules:
```

yamllint (1.35.1)

[error] duplication of key "pull_request_rules" in mapping

(key-duplicates)